scarb/src/ops/fmt.rs

```diff
@@ -66,6 +71,28 @@ pub fn format(opts: FmtOptions, ws: &Workspace<'_>) -> Result<bool> {
     Ok(result)
 }
 
+fn format_single_file(
+    target_file: &Path,
+    opts: &FmtOptions,
+    ws: &Workspace<'_>,
+    all_correct: &AtomicBool,
+) -> Result<bool> {
+    let config = FormatterConfig::default();
```

Comment:
This should also do 
```
        let mut config = FormatterConfig::default();
        if let Some(overrides) = package.tool_metadata("fmt") {
            config = toml_merge(&config, overrides)?;
        }
```
To allow setting formatting opts through scarb manifest.

Reply:
what should be the package passed here? do I need to iterate over all packages and infer from the provided path?

Reply:
Yes, that's what I would do 
```
        let target_file = target_file.canonicalize_utf8()?;
        let pkg = ws
            .members()
            .find(|member| target_file.starts_with(member.root()));
```

Reply:
thanks, did this